services/compute/serverless-computing/controls.yaml

```diff
@@ -0,0 +1,52 @@
+common_controls:
+  - CCC.C01  # Prevent unencrypted requests
+  - CCC.C02  # Ensure data encryption at rest for all stored data
+  - CCC.C03  # Implement multi-factor authentication (MFA) for access
+  - CCC.C04  # Log all access and changes
+  - CCC.C05  # Prevent access from untrusted entities
+  - CCC.C06  # Prevent deployment in restricted regions
+  - CCC.C07  # Alert on non-human enumeration
+  - CCC.C09  # Prevent tampering, deletion, or unauthorized access to access logs
+  - CCC.C10  # Prevent data replication to destinations outside of defined trust perimeter
+
+controls:
+  - id: CCC.SvlsComp.C01  # Require VPC Connector for Serverless Functions
+    title: Require VPC Connector for Serverless Functions
```

Comment:
VPC Connector is a GCP-specific term. Control definition should be common across all GCP, Azure, AWS

Reply:
Adjusted  to Private Endpoint